docs/getting-started.md

```diff
@@ -63,10 +63,10 @@ You can use Ubuntu 20.04 LTS inside Windows through Windows Subsystem for Linux,
 4. Click the `Fork` button in the top-right to fork the repository ([click here to learn about Forks](https://help.github.com/en/articles/fork-a-repo))
    1. Click on your user when prompted
    2. You should be automatically redirected to your new fork
-5. Clone your fork of the repository (you can put it wherever you want)
-   1.  Eg. `git clone https://github.com/<your-username>/Software.git`
-      1. You can find this link under the green `Clone or Download` button on the main page of the Software repository
-   2. We recommend cloning with SSH if you don't like typing your username and password all the time. Instructions can be found [here](https://help.github.com/articles/connecting-to-github-with-ssh/).
+5. Clone your fork of the repository. As GitHub is forcing users to stop using usernames and passwords, we will use the SSH link. (you can put it wherever you want)
+   1.  Eg. `git clone git@github.com:<your_username>/Software.git`
+      1. You can find this link under the green `Clone or Download` button on the main page of the Software repository, under the SSH tab
+   2. To connect to GitHub using SSH, if not setup prior, you will need to add an SSH key to your GitHub account. Instructions can be found [here](https://help.github.com/articles/connecting-to-github-with-ssh/).  For each computer you contribute to GitHub with, you will need an additional SSH Key pair linked to your account.
```

Comment:
Might be better to direct here https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent. Saves a click

Reply:
Also might be worth adding that if you've already cloned with an alternative method you need to update your remote to use ssh